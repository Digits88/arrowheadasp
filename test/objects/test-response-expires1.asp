This page expires in ten minutes.

<% Response.Expires = 10 %>

Response.Expires = <%= Response.Expires %><br>
Response.ExpiresAbolute = <%= Response.ExpiresAbsolute %><br>
