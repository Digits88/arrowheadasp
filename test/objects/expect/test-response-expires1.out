This page expires in ten minutes.



Response.Expires = 10<br>
Response.ExpiresAbolute = ^[0-9/ :APM]+$<br>
